lib/common/presentation/views/visit_us.dart/visit_us.dart

```diff
@@ -0,0 +1,471 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
```

Comment:
Changes in regards to using the theme and string constants need to be made for this file. Refer to other PR comments.